x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/downsample/DownsampleActionSingleNodeTests.java

```diff
@@ -180,47 +181,80 @@ public void setup() {
         if (randomBoolean()) {
             settings.put(IndexMetadata.SETTING_INDEX_HIDDEN, randomBoolean());
         }
-        assertAcked(
-            client().admin()
-                .indices()
-                .prepareCreate(sourceIndex)
-                .setSettings(settings.build())
-                .setMapping(
-                    FIELD_TIMESTAMP,
-                    "type=date",
-                    FIELD_DIMENSION_1,
-                    "type=keyword,time_series_dimension=true",
-                    FIELD_DIMENSION_2,
-                    "type=long,time_series_dimension=true",
-                    FIELD_NUMERIC_1,
-                    "type=long,time_series_metric=gauge",
-                    FIELD_NUMERIC_2,
-                    "type=double,time_series_metric=counter",
-                    FIELD_LABEL_DOUBLE,
-                    "type=double",
-                    FIELD_LABEL_INTEGER,
-                    "type=integer",
-                    FIELD_LABEL_KEYWORD,
-                    "type=keyword",
-                    FIELD_LABEL_TEXT,
-                    "type=text",
-                    FIELD_LABEL_BOOLEAN,
-                    "type=boolean",
-                    FIELD_METRIC_LABEL_DOUBLE, /* numeric label indexed as a metric */
-                    "type=double,time_series_metric=counter",
-                    FIELD_LABEL_IPv4_ADDRESS,
-                    "type=ip",
-                    FIELD_LABEL_IPv6_ADDRESS,
-                    "type=ip",
-                    FIELD_LABEL_DATE,
-                    "type=date,format=date_optional_time",
-                    FIELD_LABEL_KEYWORD_ARRAY,
-                    "type=keyword",
-                    FIELD_LABEL_DOUBLE_ARRAY,
-                    "type=double"
-                )
-                .get()
-        );
+
+        XContentBuilder mapping = jsonBuilder().startObject()
```

Comment:
oooof this is a bit unreadable. I know you are just reformatting, but can you add blocks or something so I can follow this? Sorry for the trouble.

Reply:
No trouble at all. Fixing this